src/content/docs/apm/agents/ruby-agent/configuration/distributed-tracing-ruby-agent.mdx

```diff
@@ -14,35 +14,28 @@ Distributed tracing allows you to see the entire journey of your requests throug
 
-* Infinite Tracing (tail-based sampling): Our cloud-based service accepts all your traces and then sorts through them to find the most important. After you set up standard tracing, we recommend you add this option because it analyzes all of your traces and gives you configuration options to sample the traces that matter most to you.
+* Infinite Tracing (tail-based sampling): Our cloud-based service accepts all your traces and then sorts through them to find the most important. Infinite tracing analyzes all of your traces and gives you configuration options to sample the traces that matter most to you.
 
-All the steps you need to get started with distributed tracing are right here:
+Whether you just want to try out standard distributed tracing (head-based sampling) or also want to set up Infinite Tracing (tail-based sampling), you need to start by setting up standard tracing. 
 
-* [New Ruby agents:](#new-agents) Steps for head- and tail-based sampling for new agent installations
-* [Older Ruby agents:](#older-agents) Tracing options if you have older Ruby agents
-* [Manual instrumentation:](#manual-instrumentation) Tips if automatic instrumentation doesn't work
+## Standard distributed tracing [#quick-start-apm]
 
-## New Ruby agents [#new-agents]
-
-Whether you just want to try out standard distributed tracing (head-based sampling) or also want to set up Infinite Tracing (tail-based sampling), you need to start by setting up standard tracing. We'll get you through the APM agent installation to get head-based sampling going. After that, you can set up Infinite Tracing, which is optional but recommended.
-
-### Install an agent to get standard distributed tracing [#quick-start-apm]
-
-This is the best approach to set up [standard distributed tracing](/docs/distributed-tracing/concepts/how-new-relic-distributed-tracing-works/#head-based) if you haven't installed any APM agents for your services yet, or if you want to instrument additional services.
-
-If you already have some services instrumented with this APM agent, and you want to include them in distributed tracing, you'll need to manually enable distributed tracing for each service. See [Options for older Ruby agents](#older-agents).
+This is the best approach to set up [standard distributed tracing](/docs/distributed-tracing/concepts/how-new-relic-distributed-tracing-works/#head-based) if you haven't installed any APM agents for your services yet.
 
 <Callout variant="tip">
-  You'll need a New Relic account to set up distributed tracing. If you don't already have one, you can quickly [create a free account](https://newrelic.com/signup).
+  When you install the New Relic Ruby agent, standard distributed tracing is turned on by default. If you prefer to turn it off, see our [configuration guide](/docs/apm/agents/ruby-agent/configuration/ruby-agent-configuration).  
```

Comment:
The same as the others. I'd add the link to the installation of the agent: /docs/apm/agents/ruby-agent/installation/install-new-relic-ruby-agent/

I'd add the anchor to the link:
/docs/apm/agents/ruby-agent/configuration/ruby-agent-configuration/#distributed-tracing

Reply:
@nbaenam just like with the Go agent, I don't want the reader to start the installation yet. I tell them how to do that in the next few steps.